languages/fau-degree-program-output-de_DE.po

```diff
@@ -1,17 +1,18 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: FAU Degree Program Output 0.0.4\n"
-"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/fau-degree-program-sync-and-output\n"
-"POT-Creation-Date: 2023-07-26T07:16:49+00:00\n"
-"PO-Revision-Date: 2023-08-02 09:33+0200\n"
+"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/FAU-Studium-"
```

Comment:
Why do tools always believe we publish the packages? 🙄
If it's a 5-minutes fix, can we tell PoEdit not to create this? Otherwise, just ignore.

Reply:
I think the problem here is different, the `Report-Msgid-Bugs-To` header is generated when we create a `.pot` file using the `wp i18n make-pot` command. As I see [there is](https://developer.wordpress.org/cli/commands/i18n/make-pot/) an option [--headers=<headers>], so we can add custom header:

```
wp i18n make-pot --headers='{"Report-Msgid-Bugs-To":"https://github.com/RRZE-Webteam/FAU-Studium-Embed"}'
```